.circleci/config.yml

```diff
@@ -359,7 +367,75 @@ jobs:
 
 workflows:
   version: 2
-  add_branch:
+  deploy_branch_approval_required:
```

Comment:
Is this workflow (and `branch_approval_required` parameter) just for testing or needed?

Reply:
@davidread It is just for testing and needs to be removed